src/Symfony/Component/Workflow/Tests/StateMachineTest.php

```diff
@@ -38,4 +41,76 @@ public function testCanWithMultipleTransition()
         $this->assertTrue($net->can($subject, 't2'));
         $this->assertTrue($net->can($subject, 't3'));
     }
+
+    public function testBuildTransitionBlockerList()
+    {
+        $definition = $this->createComplexStateMachineDefinition();
+
+        $net = new StateMachine($definition);
+        $subject = new \stdClass();
+
+        // If you are in place "a" you should be able to apply "t1"
+        $subject->marking = 'a';
+        $this->assertTrue($net->buildTransitionBlockerList($subject, 't1')->isEmpty());
+        $subject->marking = 'd';
+        $this->assertTrue($net->buildTransitionBlockerList($subject, 't1')->isEmpty());
+
+        $subject->marking = 'b';
+        $this->assertFalse($net->buildTransitionBlockerList($subject, 't1')->isEmpty());
+    }
+
+    public function testBuildTransitionBlockerListWithMultipleTransition()
+    {
+        $definition = $this->createComplexStateMachineDefinition();
+
+        $net = new StateMachine($definition);
+        $subject = new \stdClass();
+
+        // If you are in place "b" you should be able to apply "t1" and "t2"
```

Comment:
comment is not consistent with the following code

Reply:
I did not notice that while copying and modifying testCanWithMultipleTransition. There is the same inconsistency.